packages/python/plotly/plotly/express/_core.py

```diff
@@ -270,8 +272,14 @@ def make_trace_kwargs(args, trace_spec, trace_data, mapping_labels, sizeref):
     fit_results : dict
         fit information to be used for trendlines
     """
+    trace_data: nw.DataFrame
+    df: nw.DataFrame = args["data_frame"]
+
     if "line_close" in args and args["line_close"]:
-        trace_data = pd.concat([trace_data, trace_data.iloc[:1]])
+        trace_data = nw.maybe_reset_index(
```

Comment:
@FBruzzesi Why is `maybe_reset_index` required here?

Reply:
You are right, `pd.concat` has `ignore_index=False` as default 👌